policyengine_us/variables/gov/aca/slspc/family_tier_multiplier.py

```diff
@@ -0,0 +1,109 @@
+from policyengine_us.model_api import *
+
+
+class family_tier_multiplier(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "ACA family tier multiplier for premium calculation"
+    unit = "/1"
+    definition_period = MONTH
+
+    def formula(tax_unit, period, parameters):
```

Comment:
If I am understanding this right, this could be as simple as:

```
family_category = tax_unit("slcsp_family_tier_category", period)
p = parameters(period)....
state_code = tax_unit.household("state_code", period)
in_ny = state_code == state_code.possible_values.NY
return where(in_ny, p.ny[family_category], p.vt[family_category])
```